impl/src/main/java/org/ehcache/impl/internal/store/heap/OnHeapStore.java

```diff
@@ -959,9 +984,9 @@ public void invalidateAllWithHash(long hash) throws StoreAccessException {
   private ValueHolder<V> getValue(final ValueHolder<V> cachedValue) {
     if (cachedValue instanceof Fault) {
       return ((Fault<V>)cachedValue).get();
-    } else {
-      return cachedValue;
     }
+
+    return cachedValue;
```

Comment:
Code style preference ... but I'd rather stick with the explicit else. If someone was to screw up the if branch to no longer return, this would compile fine. The previous version would fail saying that there is a missing return statement.